crates/proc_macro_srv/Cargo.toml

```diff
@@ -10,9 +10,9 @@ edition = "2018"
 doctest = false
 
 [dependencies]
-goblin = "0.2.1"
 libloading = "0.6.0"
 memmap = "0.7"
+object = { version = "0.22", git = "https://github.com/gimli-rs/object/" }
```

Comment:
Why use a git dependency?

Reply:
To pull in https://github.com/gimli-rs/object/pull/261 for https://github.com/gimli-rs/object/issues/222.